### PR TITLE
Add docker things for documentation

### PR DESCRIPTION
**Pull request by tcoupin**
### Description

Add `docs` target to makefile to test documentation with docker.

### CODE REVIEW

**Comment by ldez** [comment]
@tcoupin any news?

**Comment by tcoupin** [comment]
Sorry, new home, new job => no time :)
I have amended my commit. It's ok for you ?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -147,6 +147,26 @@ ok      _/home/user/go/src/github/containous/traefik    0.004s
...
 
+### Method 1: `Docker` and `make`
+
+You can test documentation using the `docs` target.
+
+```bash
+$ make docs
+docker build -t traefik-docs -f docs.Dockerfile .
+# […]
+docker run  --rm -v /home/user/go/github/containous/treafik:/mkdocs -p 8000:8000 traefik-docs mkdocs serve
```
Comment (Juliens) [inline CONTRIBUTING.md]: treafik => traefik

**Comment by tcoupin** [comment]
Too fast ctrl-c ctrl-v... Fix made

File: docs.Dockerfile [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+RUN apk --update upgrade \
+&& apk --no-cache --no-progress add py-pip \
+&& rm -rf /var/cache/apk/*
+
+RUN mkdir /mkdocs
+
+WORKDIR /mkdocs
+COPY requirements.txt /mkdocs
+
```
Comment (nmengin) [inline docs.Dockerfile]: You don't have to create mkdocs directory with the instruction `RUN mkdir /mkdocs`, the `COPY` instruction create the directory automatically.

**Pull request merged**